src/translation.cpp

```diff
@@ -64,7 +64,13 @@ void Translations::loadTranslation(const std::string &data)
 			line.resize(line.length() - 1);
 
 		if (str_starts_with(line, "# textdomain:")) {
-			textdomain = utf8_to_wide(trim(str_split(line, ':')[1]));
+			auto splitted = str_split(line, ':');
```

Comment:
The past form of "split" is in fact "split".

Reply:
I tend to prefer wrong english that is less ambiguous and more clear than correct english, at least in code.

Changed to `parts`.